_artifacts/junit__02.xml

```diff
@@ -0,0 +1,73 @@
+<?xml version="1.0" encoding="UTF-8"?>
+  <testsuite name="cert-manager e2e suite" tests="1" failures="0" errors="0" time="25.414">
+      <testcase name="[Conformance] Certificates with issuer type VaultAppRole ClusterIssuer should issue an ECDSA, defaulted certificate for a single distinct dnsName" classname="cert-manager e2e suite" time="0">
```

Comment:
`_artifacts` shouldn't be here 😬

Reply:
aaaaaaaaaaaargh.... we really need to .gitignore that